hack/tool/tackle

```diff
@@ -134,7 +145,7 @@ def cmdWanted(args, action):
 
 class TackleTool:
     # TYPES order matters for import/upload to Tackle2
-    TYPES = ['tagtypes', 'tags', 'jobfunctions', 'stakeholdergroups', 'stakeholders', 'businessservices', 'applications', 'proxies', 'dependencies', 'assessments', 'assessments--assessment-risk', 'assessments--confidence', 'reviews', 'proxies', 'identities']
```

Comment:
'assessments--assessment-risk', 'assessments--confidence' types were left from past, it is not needed - such data are calculated from the main assessment questionnaire. (and these types collections were empty even in 2.0 version release)